tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -559,20 +559,20 @@ static PyObject * THPVariable_to(PyObject* self, PyObject* args, PyObject* kwarg
 {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
-    "to(Device device, ScalarType dtype=None)",
+    "to(Device device, ScalarType dtype=None, bool non_blocking=False)",
     "to(ScalarType dtype)",
```

Comment:
nit: I'd just throw non_blocking on here too even though it doesn't do anything (a lot of combinations don't do anything already).  It just makes it easier to write functions that take devices and/or dtypes without having to move the error checking up a level.